app/bundles/FormBundle/Tests/Helper/FormFieldHelperTest.php

```diff
@@ -27,6 +28,120 @@ protected function setUp(): void
         $this->fixture = new FormFieldHelper($translatorMock, $validatorMock);
     }
 
+    /**
+     * @dataProvider validFieldDataProvider
+     */
+    public function testPopulateDateTimeValues(string $type, string $defaultValue, string $expected, string $msg): void
+    {
+        $form = new Form();
+        $form->setName('mautic');
+
+        $field = new Field();
+        $field->setType($type);
+        $field->setDefaultValue($defaultValue);
+        $field->setAlias('fieldalias');
+        $form->addField(1, $field);
+
+        $fieldHtml = "<input value=\"{$field->getDefaultValue()}\" id=\"mauticform_input_mautic_fieldalias\">";
+
+        $actual = $this->fixture->populateDateTimeValues($form, $fieldHtml);
+
+        $this->assertEquals($expected, $actual, $msg);
+    }
+
+    /**
+     * @return array[]
```

Comment:
Try
```suggestion
     * @return array<array<string,string>>
```
to make STAN happy

Reply:
done